### PR TITLE
Add OS detection to make plugin work on Windows

### PR DESCRIPTION
**Pull request by andreas-schilling**
fix #2

### CODE REVIEW

File: gradle.properties [hunk 1]
```diff
@@ -1,3 +1,4 @@
-version=0.0.2-SNAPSHOTorg.gradle.daemon   = true
+version=0.0.2-SNAPSHOT
```
Comment (rwinch) [inline gradle.properties]: Can you please put this is a separate PR since it is a separate issue? Otherwise we won't have an appropriate changelog

**Review by rwinch — CHANGES_REQUESTED** [review]

File: gradle.properties (hunk 1 above)
Comment (andreas-schilling) [reply gradle.properties]: Oh yes, sure.

**Comment by rwinch** [comment]
Thanks for the update. I went ahead and squashed the commits and merged.
